search-app/src/cmr/search/site/routes.clj

```diff
@@ -2,19 +2,51 @@
   "This namespace is the one responsible for the routes intended for human
   consumption."
   (:require
-   ;; Third-party libs
-   [compojure.core :refer :all]
-   [ring.swagger.ui :as ring-swagger-ui]
+    ;; Third-party libs
+    [compojure.core :refer :all]
+    [ring.util.response :refer [redirect]]
+    [ring.swagger.ui :as ring-swagger-ui]
 
-   ;; CMR libs
-   [cmr.collection-renderer.api.routes :as collection-renderer-routes]
-   [cmr.common.api.context :as context]
-   [cmr.common-app.api-docs :as api-docs]))
+    ;; CMR libs
+    [cmr.collection-renderer.api.routes :as collection-renderer-routes]
+    [cmr.common.api.context :as context]
+    [cmr.common-app.api-docs :as api-docs]
+    [cmr.search.site.pages :as pages]))
+
+(defn- redir-url
```

Comment:
Okay, that's what I ended up doing -- but in order for this to work locally (dev deploy), I've had to update the system data structure in `dev-system.user` so that the search app gets host and protocol set in its `public-conf`. Hacktastic. Really want to find out the CMR best practice for this ...

Will also need to find out what the system integration tests best practice is for that, since that's the next thing that will break :-)